code/modules/mob/abstract/observer/observer.dm

```diff
@@ -186,9 +186,10 @@ Works together with spawning an observer, noted above.
 		return
 
 	if(following)
-		if(!iscarbon(following)) //If they are following something other than a carbon mob, teleport them
+		if(!isliving(following)) //If they are following something other than a living mob, teleport them
```

Comment:
This is too broad, as it covers simple mobs aswell.

Reply:
I don't believe it's an issue; for players to be able to comprehend anything you'd have to have the mob there, be able to find it in the follow list, have it be in some place where following reveals anything interesting or important *and* that place would have to be lit up for you to see anything (since even ghost sight is limited on those Z-levels).

With that being said I can easily add `!isanimal(A)`. Can you think of anything else that could be an issue?